lib/tabula.js

```diff
@@ -52,7 +52,17 @@ function sortArrayOfObjects(items, fields, options) {
 
     function cmp(a, b) {
         for (var i = 0; i < fields.length; i++) {
-            var field = fields[i];
+            var obj = fields[i];
+            if (typeof (obj) !== 'object') {
+                obj = {
+                    field: obj
+                };
+            }
+            assert.object(obj, 'obj');
+            assert.string(obj.field, 'obj.field');
+            assert.optionalFunc(obj.comparator, 'obj.comparator');
```

Comment:
I'd consider providing a "key function" argument instead of a "compare function". The former is what Python's list sort provides: https://docs.python.org/3.3/library/stdtypes.html?highlight=sort#list.sort Basically the idea is that the given `key` function is used to get a comparable value from the array elements for comparison. Sometimes this makes it simpler for the user. (For Python's implementation, it can make the list sort much faster because less computation is necessary when repeatedly comparing values in the process of sorting the array. Anyway, the perf benefit doesn't apply to node-tabula here, so this is more about the UX.) Are you able to show your TRITON-24 code that would use this as a sanity test for the UX diff?